app/models/saved_claim.rb

```diff
@@ -91,8 +91,24 @@ def form_must_be_string
   def form_matches_schema
     return unless form_is_string
 
-    JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM], parsed_form).each do |v|
-      errors.add(:form, v.to_s)
+    schema = VetsJsonSchema::SCHEMAS[self.class::FORM]
+
+    schema_errors = JSON::Validator.fully_validate_schema(schema, { errors_as_objects: true })
+    clear_cache = false
+    unless schema_errors.empty?
```

Comment:
🎨 IMHO cleaner to avoid the double-negative in conditionals like this:

```suggestion
    if schema_errors.any?
```